client/sites-dashboard-v2/index.tsx

```diff
@@ -120,24 +134,26 @@ const SitesDashboardV2 = ( {
 		}
 	}, [ dataViewsState.selectedItem ] );
 
-	// Update URL with search param on change
+	// Update URL with view control params on change.
 	useEffect( () => {
 		const queryParams = {
 			search: dataViewsState.search?.trim(),
-			status: statusSlug === 'all' ? undefined : statusSlug,
+			status: statusSlug === DEFAULT_STATUS_GROUP ? undefined : statusSlug,
+			page: dataViewsState.page === 1 ? undefined : dataViewsState.page,
```

Comment:
There is a pre-existing issue where "page" resets to 1 on reloading of the component regardless of what we initialize the state as. Im not yet sure why this happens, but it may be part of the initialization of the gutenberg data components. This makes keeping track of it in the url seem useless currently. Im tempted to just remove it from the queryParams entirely for now. 🤔  Thoughts?

Reply:
@Addison-Stavlo if it's useless then I agree that it's better to remove it for now and leave the URL less cluttered. We can always bring it back later 🙂

Reply:
Thanks for the input @taipeicoder ! Il remove it for now. I was wondering if leaving it around would bug us enough to address the underlying cause sooner, but it also [doesn't seem like an impactful feature for initial release and something we can follow up on](https://github.com/Automattic/wp-calypso/pull/89698#discussion_r1576880218).

Il update this and remove.